azure/create-cluster.sh

```diff
@@ -43,7 +43,7 @@ current_branch_name=$(git branch --show-current)
 export BRANCH=${CIRCLE_BRANCH:=$current_branch_name}
 
 # get local public ip 
-local_public_ip=$(curl https://ipinfo.io/ip)
+local_public_ip=$(host myip.opendns.com resolver1.opendns.com | grep "myip.opendns.com has" | awk '{print $4}')
```

Comment:
While the lhs one only returns the ip address, the rhs one appends an "\n" to the end, so why do we want the rhs one ?

Reply:
Actually, @naisila tested and this command does not work either.
MSVPN causes pinging both of these sites to timeout.
If I remember correctly, the hack to get this to work is to turn on MSVPN after running the script. Just before ssh-ing into the cluster.
At the very least the README needs to be updated, though I would prefer if we can actually fix the pinging.
Will look some more into this.

Reply:
Apparently MSVPN is not the only culprit here:
https://github.com/microsoft/WSL/issues/4517
WSL + VPN actually causes this to hang.
Networking in WSL is tricky.

Reply:
oh really ? okay then :)

Reply:
I'm wondering if \n at the end could cause any issues then, but I don't think so --let's see